src/test/groovy/com/intuit/graphql/orchestrator/batch/BatchResultTransformerSpec.groovy

```diff
@@ -0,0 +1,140 @@
+package com.intuit.graphql.orchestrator.batch
+
+import static graphql.execution.MergedField.newMergedField;
+import static graphql.language.Field.newField;
+
+import graphql.GraphQLError;
+import graphql.GraphqlErrorBuilder;
+import graphql.execution.DataFetcherResult;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.DataFetchingEnvironmentImpl;
+import helpers.BaseIntegrationTestSpecification
+
+class BatchResultTransformerSpec extends BaseIntegrationTestSpecification {
```

Comment:
same.. we can use `Specification` of groovy instead of `BaseIntegrationTestSpecification`.  Some unit test classes as well might need just to extend the `Specification`